patches/chrome-test-BUILD.gn.patch

```diff
@@ -1,13 +1,13 @@
 diff --git a/chrome/test/BUILD.gn b/chrome/test/BUILD.gn
-index 63be9ca9369fd761a4ca92a56cbf81a6ec3d56ed..0e81a26632d26450c2fdd7839214cfa52a614efb 100644
+index d637b306110b4adfb1f4b85be3491949a98803c5..4f4b52000f56c4b41963df094cac280fc3510802 100644
 --- a/chrome/test/BUILD.gn
 +++ b/chrome/test/BUILD.gn
-@@ -470,6 +470,8 @@ test("browser_tests") {
-     "//third_party/blink/public:buildflags",
-   ]
+@@ -591,6 +591,8 @@ if (!is_android) {
+       "//v8",
+     ]
  
-+  deps += [ "//brave/test:brave_browser_tests_deps", ]
++    deps += [ "//brave/test:brave_browser_tests_deps", ]
```

Comment:
also if you come across existing patches with empty lines added please remove them

Reply:
for future reference not follow-up ^^

Reply:
Addressed in 8ebe58a